doc/source/loggers.rst

```diff
@@ -13,24 +13,106 @@ by caproto, below. Because recalling the name is not always convenient, certain
 objects in caproto's Python API expose a ``log`` attribute, such as ``pv.log``,
 ``pv.context.log`` and ``pv.circuit_manager.log``.
 
+You could find python logging flow here.
+https://docs.python.org/2/_images/logging_flow.png
+
+
 Useful snippets
 ===============
 
-To turn on maximal logging (usually way too much information!) use
+Make sure you understand how level control loggging in logger.
+and handler. Both has method setLevel(...) which allow you do logger.setLevel(...) or
+handler.setLevel(...). You may only have one logger in your package. But, mutiple
+handlers could be added to one logger. Logger's level influence all handler. So the
+effective level is the intersection of logger's level and handler's level.
+
+
+Here INFO will effect
 
 .. code-block:: python
 
     import logging
```

Comment:
The intention of this "Useful snippets" section is to provide short, copy-pasteable examples that solve specific common situations, such as showing all DEBUG messages, showing all INFO message, or showing DEBUG messages above a specific PV.

Reply:
This code is response to last item of check list. The PV and level setting for filter is showed in this block https://github.com/NSLS-II/caproto/pull/427/files#diff-3587a713fc2402099edc2661384825ddR102